go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/attestantio/go-builder-client
 go 1.18
 
 require (
-	github.com/attestantio/go-eth2-client v0.15.1
+	github.com/attestantio/go-eth2-client v1.15.2
```

Comment:
This version has been retracted and should not be used.  0.15.1 is the latest correct version.